hanadb_exporter/secret_manager.py

```diff
@@ -0,0 +1,32 @@
+import boto3
+import requests
+import json
+from botocore.exceptions import ClientError
+
+def get_db_credentials(secret_name):
+    ec2_info_url = "http://169.254.169.254/latest/dynamic/instance-identity/document"
```

Comment:
This value, as a constant, should be name in capital letters.

Reply:
agree

Reply:
done https://github.com/SUSE/hanadb_exporter/pull/97